examples/slack/README.md

```diff
@@ -0,0 +1,195 @@
+# Slack example
+
+This `repo.yaml` file contains a workflow specification to send Infracost cost estimates to Slack.
+
+Slack message blocks have a 3000 char limit so the Infracost CLI automatically truncates the middle of `slack-message` output format.
+
+## Table of Contents
+
+* [Running with GitHub](#running-with-github)
+* [Running with GitLab](#running-with-gitlab)
+* [Running with Azure Repos](#running-with-azure-repos)
```

Comment:
Added ToC and the title on line 1. If it's 👍 I'll go over other examples and do the same.